src/main/java/frc/robot/Constants.java

```diff
@@ -25,31 +33,79 @@
  */
 public final class Constants {
 
-  public static final double ROBOT_MASS = (65) * 0.453592; // 32lbs * kg per pound
-  public static final Matter CHASSIS = new Matter(new Translation3d(0, 0, Units.inchesToMeters(8)), ROBOT_MASS);
-  public static final double LOOP_TIME = 0.13; // s, 20ms + 110ms sprk max velocity lag
+    public static final double ROBOT_MASS = (65) * 0.453592; // 32lbs * kg per pound
+    public static final Matter CHASSIS = new Matter(new Translation3d(0, 0, Units.inchesToMeters(8)), ROBOT_MASS);
+    public static final double LOOP_TIME = 0.13; // s, 20ms + 110ms sprk max velocity lag
+
+    public static final class Auton {
+
+        public static final PIDConstants TranslationPID = new PIDConstants(3, 0.0, 0.0);
+        public static final PIDConstants angleAutoPID = new PIDConstants(2, 0, 0);
+
+        public static final double MAX_ACCELERATION = 2;
+    }
+
+    public static final class Drivebase {
+
+        // Hold time on motor brakes when disabled
+        public static final double WHEEL_LOCK_TIME = 10; // seconds
+    }
+
+    public static class OperatorConstants {
+
+        // Joystick Deadband
+        public static final double LEFT_X_DEADBAND = 0.01;
+        public static final double LEFT_Y_DEADBAND = 0.01;
+        public static final double RIGHT_X_DEADBAND = 0.01;
+        public static final double TURN_CONSTANT = 6;
+        public static final int JOYSTICK_PORT = 1;
+    }
+
+    public static class FieldConstants {
+        public static final double length = Units.feetToMeters(54);
+        public static final double width = Units.feetToMeters(27);
+    }
+
+    public static final class VisionConstants {
+        private static double X = (53.0 / 100) / 2;
+        private static double Y = (53.0 / 100) / 2;
+        private static double Z = (36.0 / 100);
```

Comment:
I would suggest naming these a bit more descriptive and, including the units in the name, something like:
```
  Constants.ROBOT_X_SIZE_M = 0.53 // robot size along the X axis in meters
  Constants.ROBOT_Y_SIZE_M = 0.53 // robot size along the Y axis in meters
  Constants.ROBOT_Z_SIZE_M = 0.36 // robot height (size along the Z axis) in meters

  VisionConstants.ROBOT_HALF_X_SIZE_M = Constants.ROBOT_X_SIZE_M / 2
  VisionConstants.ROBOT_HALF_Y_SIZE_M = Constants.ROBOT_Y_SIZE_M / 2
  VisionConstants.ROBOT_Z_SIZE_M = Constants.ROBOT_Z_SIZE_M
```

..or any other names that you like , as long as they are descriptive enough.